testing/git-crypt/APKBUILD

```
@@ -0,0 +1,31 @@
+# Contributor: Laurent Arnoud <laurent@spkdev.net>
+# Maintainer: Laurent Arnoud <laurent@spkdev.net>
+pkgname=git-crypt
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="Transparent file encryption in git"
+url="https://www.agwa.name/projects/git-crypt"
+arch="all"
+license="GPL-3.0-or-later"
+depends="git"
```

Disable check since no test suite is available.
```suggestion
depends="git"
options="!check" # No test suite
```

---

Ok done in https://github.com/alpinelinux/aports/pull/7070/commits/720d0ec991726340a1bf9cb30999aa61590a8b3c